manifests/candlepin.pp

```diff
@@ -1,6 +1,6 @@
 # Constains certs specific configurations for candlepin
 class certs::candlepin (
-  $hostname               = $certs::node_fqdn,
+  $hostname               = 'localhost',
```

Comment:
Should this reflect the value of $candlepin::params::host or whatever value was giving to puppet-candlepin?

Reply:
That's a fair question. We'd handle this by whatever calls this class passing this value in. That could mean we should change (https://github.com/theforeman/puppet-katello/blob/master/manifests/candlepin.pp#L31) to be:

```
class { 'certs::candlepin':
  hostname => $candlepin::host
}
```

@ekohl am I thinking correctly here or is there some other method we'd expect?

Reply:
I was asking in reference to a comment on the related PR: 

https://github.com/theforeman/puppet-candlepin/pull/142#issuecomment-587504985

Reply:
These are the defaults, but I do feel you're on the right path. IMHO [katello::candlepin](https://github.com/theforeman/puppet-katello/blob/master/manifests/candlepin.pp) is the right place to connect these two. There we can do something like:
```puppet
include certs::candlepin
class { 'candlepin':
  host => $certs::candlepin::hostname,
}
```

Now the end users' experience will be that they can use `--certs-candlepin-hostname` to set the hostname. The alternative is to add a `$hostname` parameter to `katello::candlepin`. The benefit to that is that it feels a bit more natural for the end user but we need to look at the configuration of the various certs classes and how to expose it. In our team we've been talking at evaluating the certs usage in the installer.